include/input/keyboard.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_KEYBOARD_H
+#define LABWC_KEYBOARD_H
+
+#include <stdbool.h>
+
+struct seat;
+struct keyboard;
+struct wlr_keyboard;
+
+void keyboard_init(struct seat *seat);
+void keyboard_finish(struct seat *seat);
+
+void keyboard_set_numlock(struct wlr_keyboard *keyboard);
+void keyboard_add_keyboard_handler(struct keyboard *keyboard, struct wlr_keyboard *wlr_kb);
```

Comment:
It should likely be enough to just use the first arg to get the `wlr_keyboard`. Verify.